src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/AssetSummaryStep.tsx

```diff
@@ -57,6 +60,24 @@ export const AssetSummaryStep = ({
     return chainName
   }, [asset.chainId])
 
+  const prevAmountCryptoBaseUnit = usePrevious(amountCryptoBaseUnit)
+
+  useEffect(() => {
+    if (!isLastStep) return
+    if (
+      !(
+        amountCryptoBaseUnit &&
+        prevAmountCryptoBaseUnit &&
+        amountCryptoBaseUnit !== '0' &&
+        prevAmountCryptoBaseUnit !== '0'
+      )
+    )
+      return
+    if (amountCryptoBaseUnit === prevAmountCryptoBaseUnit) return
```

Comment:
Correct ser! Since we are at a stage where the swapper (e.g Li.Fi) / identifier (e.g LiFi PropellerSwap tool) are static, then that's actually fine. We're precisely concerned about the *rate* changing which is what we surface to users as an attention modal